src/feed-input.scss

```diff
@@ -0,0 +1,86 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-input;
+
+.#{$block} {
+  $fd-feed-input-min-height: 3rem;
+  $fd-feed-input-max-height: 20rem;
+  $fd-feed-input-max-width: calc(100vw * 2 / 3);
+  $fd-feed-input-padding-start: 1rem;
+  $fd-feed-input-padding-end: 0.5rem;
+  $fd-feed-input-thumb-margin-start: 0;
+  $fd-feed-input-thumb-margin-end: 0.5rem;
+  $fd-feed-input-btn-margin-start: 0.5rem;
+  $fd-feed-input-btn-margin-end: 0;
+
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: row;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  padding-left: $fd-feed-input-padding-start;
+  padding-right: $fd-feed-input-padding-end;
+  background-color: var(--sapField_Background);
+
+  @include fd-screen(m) {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  @include fd-screen(l) {
+    max-width: $fd-feed-input-max-width;
+  }
```

Comment:
I know those mixins can be misleading a little bit. Could you change them to modifiers? So at the bottom of file
```
&--m {
    margin-top: 1rem;
    margin-bottom: 1rem;
}

&--m {
    margin-top: 1rem;
    margin-bottom: 1rem;
}
```

Also `$fd-feed-input-max-width` is wrong, Following specs all sizes should have `max-width: 100%`

Reply:
So we can omit this statement?
![image](https://user-images.githubusercontent.com/3263744/89951072-6d18b500-dc33-11ea-87a1-472eb80067c2.png)

Reply:
Also added modifiers but left `fd-screen`. 
It's not clear for me if we should keep `fd-screen` beside modifiers or not, I saw many cases of usage `fd-screen` in the current project.

Reply:
There are some leftovers, which uses `fd-screen` @InnaAtanasova What we should do in this case?
And of course differences between fiori design guidelines and fiori3 specifications. There are 2 sources and we'll never get the true until we write to responsible designer.
CC @estelaxu3